solarforecastarbiter/datamodel.py

```diff
@@ -657,6 +677,8 @@ def __post_init__(self):
         __check_aggregate_interval_compatibility__(
             self.interval_length,
             *observations)
+        __generic_oneof__(self, 'aggregate_type', ALLOWED_AGGREGATE_TYPES)
+        __generic_oneof__(self, 'interval_label', ('beginning', 'ending'))
```

Comment:
dashboard form allows instant

Reply:
I don't think that makes sense for an aggregate, especially when we force `interval_value_type='interval_mean'` for the aggregate. ('interval_mean' because the individual observations are resampled w/ mean before whatever aggregation function is applied to all the observations)

Reply:
https://github.com/SolarArbiter/solarforecastarbiter-dashboard/issues/377